doc/classes/SkeletonIK.xml

```diff
@@ -12,12 +12,14 @@
 			<return type="Skeleton">
 			</return>
 			<description>
+                Returns the parent [Skeleton] Node that was present when SkeletonIK entered the [SceneTree]. Returns null if the parent node was not a [Skeleton] Node when SkeletonIK entered the [SceneTree].
 			</description>
 		</method>
 		<method name="is_running">
 			<return type="bool">
 			</return>
 			<description>
+                Returns [code]true[/code] if SkeletonIK is applying IK effects on continues frames to the [Skeleton] bones. Returns [code]false[/code] if SkeletonIK is stopped or [method start] was used with the [code]one_time[/code] parameter set to [code]true[/code].
```

Comment:
"continues frames" sounds weird, what does this refer to exactly in this context?

Reply:
Could use a more common word, I mean every frame that is rendered after the current frame but NOT the current frame.

Reply:
Maybe "Returns [code]true[/code] if SkeletonIK is configured to apply IK effects in future frames to the [Skeleton] bones."
Still not super clear but I'm not very familiar with the API.

Alternatively, what happens when you are in a frame where SkeletonIK *is* applying IK effects? Is `is_running()` then `false`, or is it also `true`? If true, maybe what needs to be stated is that it will be `true` after calling `start()` but will only take effect in the next frame?

Reply:
`is_running()` only checks if SkeletonIK `internal_processing` is `true `.

SkeletonIK `internal_processing `only gets enabled when a user uses start() without a bool parameter but the IK effect will only be applied on next frame and process call.